FCO.tex

```diff
@@ -296,65 +294,62 @@ \section{Plutonium stacking effect}
   \subfigure[E3'' stream flow.\label{fig:E3_LII}] 
       {\epsfig{figure=img/C_1_3_W_LII_E3s.png,width=0.48\textwidth}}
 
-  \caption{ Evolution of the J1'' and E3'' stream flow accordingly to the size
+  \caption{ Evolution of the J1'' and E3'' stream flow according to the size
     of initial inventory. \label{fig:LII} } 
   \end{figure}
 
-As observed on Figure \ref{fig:LII}, the decay of the $^{241}$Pu has a direct
-impact on the different contribution of E3'' and J1'' stream. The model trying
+As observed in Figure \ref{fig:LII}, the decay of the $^{241}$Pu has a direct
+impact on the different contributions of the E3'' and J1'' streams. The model is trying
 to balance the loss of reactivity of the J1'' stream by increasing the amount of
-E3'' stream in the mix. The variation can be up to $10\%$ in the worse case
-after 100y.
+E3'' stream in the mix. The variation can be up to $10\%$ after 100y in the worst case.
 
 \section{Discussion}
 
-In conclusion, this study has show the impact of different way to model
-the MOX fuel fabrication considering decay or not.
-The Pu-equivalence theory reach an equilibrium (in the mixing ratio of E3'' and
-J1'' stream) very close to the fix mixing ratio, when the decay is not activate.
-This was expected: the Pu-equivalent model try to build a MOX fuel with the same
-initial reactivity as the one used in the fix mixing ratio calculation, so it
+In conclusion, this study has shown the impact of different ways to model
+the MOX fuel fabrication, including whether or not decay is considered.
+The Pu-equivalence theory reaches an equilibrium (in the mixing ratio of E3'' and
+J1'' stream) very close to the fixed mixing ratio when the decay is not activated.
+This was expected: the Pu-equivalent model tried to build a MOX fuel with the same
+initial reactivity as the one used in the fixed mixing ratio calculation, so it
 used the same mixing ratio\dots  
 
-Nevertheless we can observe a variation of $20\%$ when taking into account the
-decay process in the contribution of E3'' and J1'' stream. This variation is a
+Nevertheless we can observe a variation of $20\%$ in the contribution of E3'' and J1'' streams  when taking the
```

Comment:
a variation of 20% in what? (I reordered the sentence based on my guess, but it may be wrong now)